projectroles/forms.py

```diff
@@ -957,6 +957,28 @@ def clean(self):
         except ProjectInvite.DoesNotExist:
             pass
 
+        # Local users check
+        user_email = self.cleaned_data.get('email')
+        domain = user_email[user_email.find('@') + 1 :]
+        allow_local_users = getattr(
+            settings, 'PROJECTROLES_ALLOW_LOCAL_USERS', False
```

Comment:
A very minor thing: Both `PROJECTROLES_ALLOW_LOCAL_USERS` and `ENABLE_SAML` are mandatory settings, so we can refer to them directly via `settings.*` and simplify this a bit.

Other than that, everything looks great!

Reply:
I tried to change it, but then tests failed( I think for overriding settings the variable should be set with getattr(). Should I change the tests somehow or let it be as is?